### PR TITLE
41). Add topic files 127 to 130

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also fixes some article files.

Please merge #18439, #18541, #18580, #18589, #18673, #18700, and #18711 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by mairaw** [comment]
Ok, this one is next for resolving merge conflicts @jameshkramer

File: docs/standard/linq/add-elements-attributes-nodes-xml-tree.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/add-elements-attributes-nodes-xml-tree.md:1]: the redirect for this page doesn't seem to be on the changes for this PR

File: docs/standard/linq/in-memory-xml-tree-modification-vs-functional-construction.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/in-memory-xml-tree-modification-vs-functional-construction.md:1]: same here

File: docs/standard/linq/maintain-name-value-pairs.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/maintain-name-value-pairs.md:1]: and here

File: docs/standard/linq/performance-chained-queries.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/performance-chained-queries.md:1]: and this one too

File: docs/standard/linq/wordprocessingml-document-styles.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
```
Comment (mairaw) [inline docs/standard/linq/wordprocessingml-document-styles.md:1]: this one also

File: docs/standard/linq/add-elements-attributes-nodes-xml-tree.md (hunk 1 above)
Comment (jameshkramer) [reply docs/standard/linq/add-elements-attributes-nodes-xml-tree.md]: The redirection file entries that are missing were in earlier PRs - 37). and 38). I resubmitted some of the files of those PRs, but not the redirection entries, in this PR to make some fixes. I think you can see the redirection entries now that I have resolved merge conflicts.

**Review by mairaw — APPROVED** [review]
LGTM. I'll merge once build finishes.

**Pull request merged**